### PR TITLE
Update scripts ordering in package.json

### PR DESCRIPTION
**Pull request by ShGKme**
**Current scripts order and types**

(📦 bundling) `build`
(📦 bundling) `dev`
(📦 bundling) `watch`
(🧪 testing) `test`
(🧪 testing) `test:watch`
(🧪 testing) `test:coverage`
(👮 lint) `lint`
(👮 lint) `lint:fix`
**(📦 bundling) `serve`** __<----- the imposter ------__
(👮 lint) `stylelint`
(👮 lint) `stylelint:fix`

**Grouped by type**

(📦 bundling) `build`
(📦 bundling) `dev`
(📦 bundling) `watch`
(📦 bundling) `serve`
(🧪 testing) `test`
(🧪 testing) `test:watch`
(🧪 testing) `test:coverage`
(👮 lint) `lint`
(👮 lint) `lint:fix`
(👮 lint) `stylelint`
(👮 lint) `stylelint:fix`

### CODE REVIEW

**Comment by ShGKme** [comment]
Also removed:
- `--config webpack.config.js` - redundant as default value
- `--env dev_server` seems to not change anything

File: package.json [hunk 1]
```diff
@@ -8,12 +8,12 @@
 		"build": "webpack --node-env production --progress",
 		"dev": "webpack --node-env development --progress",
 		"watch": "webpack --node-env development --progress --watch",
+		"serve": "npx webpack serve --node-env development --progress --allowed-hosts all",
```
Comment (Antreesy) [inline package.json:11]:
```suggestion
		"serve": webpack serve --node-env development --progress --allowed-hosts all",
```
Seems to also working proper

Comment (ShGKme) [reply package.json:11]: Thanks, missed it

**Comment by ShGKme** [comment]
Rebased - updated the commit message + remove redundant `npx`

**Review by Antreesy — APPROVED** [review]
:seal:

**Pull request merged**